### PR TITLE
ci: Add release please support

### PR DESCRIPTION
**Pull request by keelerm84**

### CODE REVIEW

File: .github/workflows/manual-publish.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    needs: [ 'build-publish' ]
+    permissions:
+      actions: read
+      id-token: write
+      contents: write
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.7.0
+    with:
+      base64-subjects: "${{ needs.build-publish.outputs.gem-hash }}"
+      upload-assets: ${{ !inputs.dry_run }}
+
```
Comment (kinyoklion) [inline .github/workflows/manual-publish.yml]: I wonder if this last line has a space or something. I get :no_entry: in my diff.

File: .github/workflows/release-please.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    permissions:
+      id-token: write # Needed if using OIDC to get release secrets.
+      contents: write # Contents and pull-requests are for release-please to make releases.
+      pull-requests: write
+    outputs:
+      release-created: ${{ steps.release.outputs.release_created }}
+      upload-tag-name: ${{ steps.release.outputs.tag_name }}
+      gem-hash: ${{ steps.publish.outputs.gem-hash}}
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```
Comment (kinyoklion) [inline .github/workflows/release-please.yml:20]: Do you want v3?

**Review by kinyoklion — APPROVED** [review]

File: .github/workflows/manual-publish.yml (hunk 1 above)
Comment (keelerm84) [reply .github/workflows/manual-publish.yml]: Yeah, trailing line with spaces. Cleaned it up.

File: .github/workflows/release-please.yml (hunk 2 above)
Comment (keelerm84) [reply .github/workflows/release-please.yml:20]: No, I'd rather keep all ruby SDKs on the same setup. Then I will change them all at once. I find that easier than doing them bit by bit.

**Pull request merged**